toolchain/check/testdata/class/no_prelude/name_poisoning.carbon

```diff
@@ -265,3 +265,21 @@ class C {
   // CHECK:STDERR:
   fn B();
 }
+
+// --- poisoned_name_in_extend.carbon
```

Comment:
```suggestion
// --- todo_fail_poisoned_name_in_extend.carbon
```
`todo_fail_` is our naming convention for tests that should eventually fail but don't yet.

Reply:
Thanks!
Done.